webui/cypress/support/step_definitions/base.steps.ts

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
Mark this as moved from `webui/e2e/src/steps/base.steps.ts`?

Reply:
tried, didn't work. see comment thread above.